index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+  <link rel="stylesheet" href="style.css">
+  <title>Flexbox</title>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="container">
+      <div class="logo">flexbox</div>
+      <ul class="nav">
+        <li>
+          <a href="#">Home</a>
+        </li>
+        <li>
+          <a href="#">About</a>
+        </li>
+        <li>
+          <a href="#">Contact</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <header class="header">
+    <div class="container">
+      <div>
+        <h1>Layout using Flexbox</h1>
+        <p>
+          This layout was building durgin the Flexbox Crash Course with Brad Traversy. He is one of the most valuable
+          people on YouTube and Udemy teaching
+        </p>
+      </div>
+      <img src="images/grid.svg">
+    </div>
```

Comment:
kindly add the alt element text in your <img>. Imagine a user's internet is very slow and not able to load the media of the website, in this situation, the alt element will help the user to understand what the media is all about.